qiskit_aer/backends/aer_simulator.py

```diff
@@ -846,36 +839,38 @@ def from_backend(cls, backend, **options):
             else:
                 description = backend.description
 
-            configuration = QasmBackendConfiguration(
-                backend_name=f"aer_simulator_from({backend.name})",
-                backend_version=backend.backend_version,
-                n_qubits=backend.num_qubits,
-                basis_gates=backend.operation_names,
-                gates=[],
-                local=True,
-                simulator=True,
-                conditional=True,
-                open_pulse=False,
-                memory=False,
-                max_shots=int(1e6),
-                coupling_map=(
+            configuration = {
```

Comment:
I suggest a dataclass for `configuration` to avoid a breaking change.
It would be nice to raise a deprecation warning to access `properties`.

```
backend  = AerSimulator()

# 0.14.2
backend.properties()  # None
backend.configuration().basis_gates # ['ccx',...]
backend.configuration()["basis_gates"]  # TypeError: 'QasmBackendConfiguration' object is not subscriptable

# this PR
backend.properties()  # AttributeError: 'AerSimulator' object has no attribute 'properties'
backend.configuration().basis_gates # AttributeError: 'dict' object has no attribute 'basis_gates'
backend.configuration()["basis_gates"]  # ['ccx',...]

```